### DISCUSSION TITLE
Playstore woes

### DISCUSSION THREAD

**Discussion by evant** [AndroidDev-social/DodoForMastodon#13] [General]
There's particular challenges with publishing to the playstore that I think it would be worth documenting so we can stay ahead of them/know how to react. Obviously we could avoid these by choosing not to (say only publish to F-Droid instead) but I feel that would be limiting.

1. **Asking for donations**. TLDR: don't. There's been plenty of examples of apps being suspended for asking for donations or linking to a page that does for not using IAP. However IAP doesn't allow donations. There's language in place for allowing linking if you are officially registered as a non-profit, but otherwise would recommend avoiding it.
2. **Content moderation policy**. This one's iffy because we can't control the policy of every instance unless we have an allowlist which kinda defeats the purpose of federation. According to [@ConnyDuck](https://chaos.social/@ConnyDuck/109304696761126681) showing the server rules on signup/login may be sufficient though it's unclear. We may also want to consider an in-app blocklist for egregious instances.
3. **Privacy Policy**. This one's easy, have one. Here's [Tusky's](https://tusky.app/privacy-policy/) for reference.
4. **Repetitive Content**. Because the app is open-source there's nothing stopping anyone else from publishing it as well. This has hit other open-source projects with repetitive content/impersonation suspensions. Not sure there's much we can do about this but open to ideas.

**Comment by evant** [comment]
Would also be curious about the Apple side if we want to publish on iOS which I know less about. One particular concern I have is explicit content which seems to be generally allowed on instances (behind a CW) but not allowed according to Apple's policies.

**Comment by evant** [comment]
Note: looks like @digitalbuddha is working on setting up a non-profit so we may be able to link out to that for donations eventually (with permission of course 😉)

https://androiddev.social/@friendlymike/109314476066026798

**Comment by digitalbuddha** [comment]
Surely! I image it is about 6months out.
